index.html

```diff
@@ -324,33 +323,34 @@
       <p>The <dfn>observe()</dfn> method instructs the user agent to
       <dfn>register the observer</dfn> and must run these steps:</p>
       <ol data-link-for="PerformanceObserverInit">
-        <li>Remove all unsupported names from the entryTypes sequence.</li>
-        <li>If the resulting '<a>entryTypes</a>' sequence is an empty sequence, abort
-          these steps. The user agent SHOULD notify developers when the steps are
-          aborted. A console warning listing ignored unsupported names in `entryTypes`
-          might be appropriate, for example.
-        </li>
-        <li>If the list of <a>registered performance observer</a> objects
-        associated with the <a>ECMAScript global environment</a> of the
-        interface object's contains a <a>registered performance observer</a>
-        that is the <a>context object</a>, replace the <a>context object</a>'s
-        `options` with <var>options</var>.
+        <li>Let <var>entry types</var> be <var>options</var> <a>entryTypes</a>
+          sequence.</li>
+        <li>Remove all unsupported types from <var>entry types</var>. The user
+          agent SHOULD notify developers if <var>entry types</var> is modified.
+          For example, a console warning listing removed types might be appropriate.</li>
+        <li>If the resulting <var>entry types</var> sequence is an empty sequence,
+          abort these steps. The user agent SHOULD notify developers when the
+          steps are aborted to notify that registration has been aborted. For
+          example, a console warning might be appropriate.</li>
+        <li>If the <a>list of registered performance observer objects</a> of
+          <a>current global object</a> contains a <a>registered performance
+          observer</a> that is the <a>context object</a>, replace its `options`,
+          with <var>options</var>.
         </li>
        <li>Otherwise, append a new <a>registered performance observer</a>
-        object to the list of <a>registered performance observer</a> objects
-        associated with the <a>ECMAScript global environment</a> of the
-        interface object, with the <a>context object</a> as `observer` and
-        <var>options</var> as the `options`.
+        object to the <a>list of registered performance observer objects</a> of
+        <a>current global object</a>, with the <a>context object</a> as
```

Comment:
Just make sure this matches the current vs relevant decision above.

Reply:
Paging @spanicker @toddreifsteck @JosephPecoraro for implementation feedback.. :)